src/app/components/Balances/index.tsx

```diff
@@ -0,0 +1,217 @@
+                suffix={<small>{denomText}</small>} 
+              />
+            </div>    
+            <Tooltip
+              overlayClassName="Balances-chart-tip"
+              title={`
+                ${stats.pendingPercent}% Pending /
+                ${stats.channelPercent}% Lightning /
+                ${stats.onchainPercent}% ${blockchainDisplayName[chain]}
+              `}
+              placement="topRight"
+              arrowPointAtCenter
+            >
+              <Progress
+                className="Balances-chart-progress"
+                percent={stats.channelPercent + stats.pendingPercent}
+                type="circle"
+                strokeColor="#7642ff"
+                successPercent={Math.max(0.1, 100 - stats.channelPercent - stats.onchainPercent)}
+                trailColor="#ff9500"
+                format={() => `${stats.spendablePercent}%`}
+              />
+            </Tooltip>
+          </div>
+          <div className="Balances-stats">
+            <Statistic 
+              title="Pending" 
+              value={parseFloat(fromBaseToUnit(stats.pendingTotal, denomination))} 
+              valueStyle={{ color: '#858585' }} 
+            />
+            <Statistic 
+              title="Lightning" 
+              value={parseFloat(fromBaseToUnit(stats.channelTotal, denomination))} 
+              valueStyle={{ color: '#7642ff' }} 
+            />
+            <Statistic 
+              title={blockchainDisplayName[chain]}
+              value={parseFloat(fromBaseToUnit(stats.onchainTotal, denomination))} 
+              valueStyle={{ color: '#ff9500' }} 
+            />
+          </div>
+          <div className="Balances-details">
+            <Tabs defaultActiveKey={stats.pendingPercent ? 'pending' : 'lightning'}>
+              <Tabs.TabPane
+                tab={<><Icon type="clock-circle"/> Pending</>}
+                key="pending"
+              >
+                <BalanceDetails 
+                  groups={stats.pendingDetails} 
+                  chain={chain}
+                  denomination={denomination}
+                  emptyMessage="All of your funds are available to spend immediately"
+                />
+              </Tabs.TabPane>
+              <Tabs.TabPane
+                tab={<><Icon type="thunderbolt"/> Lightning</>}
+                key="lightning"
+              >
+                <BalanceDetails
+                  groups={stats.channelDetails} 
+                  chain={chain}
+                  denomination={denomination}
+                  emptyMessage="Open some channels to enjoy the Lightning experience"
+                />
+              </Tabs.TabPane>
+              <Tabs.TabPane
+                tab={<><Icon type="link"/> Bitcoin</>}
+                key="on-chain"
+              >
+                <BalanceDetails 
+                  groups={stats.onchainDetails} 
+                  chain={chain} 
+                  emptyMessage="Fund you"
```

Comment:
Maybe I'm missing context but is this message supposed to be more descriptive?

Reply:
Yup.. it was probably late and I lost my train of thought.